CHANGES.rst

```diff
@@ -17,6 +17,8 @@ documentation
 
 - Removed unused ``grow`` parameter from ``outlier_detection`` docs. [#8156]
 
+- Fixed small typo in ``user_documentation`` docs. [#8178]
```

Comment:
1.13.3 is now released, so this file needs to rebased from master to get the current version and then this entry should go in the top section for 1.13.4 (unreleased).